build.gradle.kts

```
@@ -18,32 +18,45 @@ group = "org.openrewrite.recipe"
 val latest = if (project.hasProperty("releasing")) "latest.release" else "latest.integration"
 dependencies {
     api(platform("org.openrewrite:rewrite-bom:$latest"))
+
+    api("org.openrewrite:rewrite-cobol:$latest")
+    api("org.openrewrite:rewrite-javascript:$latest")
     api("org.openrewrite:rewrite-kotlin:$latest")
+    api("org.openrewrite:rewrite-polyglot:$latest")
     api("org.openrewrite:rewrite-python:$latest")
+    api("org.openrewrite:rewrite-templating:$latest")
+
     api("org.openrewrite.meta:rewrite-analysis:$latest")
+
+    api("org.openrewrite.recipe:rewrite-ai-search:$latest")
+
     api("org.openrewrite.recipe:rewrite-circleci:$latest")
     api("org.openrewrite.recipe:rewrite-cloud-suitability-analyzer:$latest")
     api("org.openrewrite.recipe:rewrite-concourse:$latest")
     api("org.openrewrite.recipe:rewrite-cucumber-jvm:$latest")
     api("org.openrewrite.recipe:rewrite-github-actions:$latest")
+    api("org.openrewrite.recipe:rewrite-hibernate:$latest")
+    api("org.openrewrite.recipe:rewrite-java-dependencies:$latest")
     api("org.openrewrite.recipe:rewrite-java-security:$latest")
     api("org.openrewrite.recipe:rewrite-java-dependencies:$latest")
```

```suggestion
    api("org.openrewrite.recipe:rewrite-java-dependencies:$latest")
    api("org.openrewrite.recipe:rewrite-java-security:$latest")
```
I suggest removing the existing rewrite-java-dependencies entry which is out of alphabetical order